ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -23,6 +24,7 @@ RUN __LLVM_VERSION__="6.0" \
  && apt-get update -q \
  && apt-get upgrade -qy \
  && apt-get install -qy \
+  --no-install-recommends \
```

Comment:
Nit: I'd prefer `apt-get install` options that aren't packages to be on the same line as `apt-get install` (just like `-qy`).